.gitignore

```diff
@@ -25,3 +25,4 @@ yarn-error.log*
 /dist
 /.idea
 /.vscode
+/**/__snapshots__/
```

Comment:
you should not gitignore __snapshots__, with this ignore there is no sense in snapshots tests because only thing that snapshot tests do  is to compare previous results with current one, so we know for sure which changes occurred and whether they should have happened